config/PXSecOnElectron.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<alg_conf>
+
+<!--
+Configuration for the PXSecOnElectron xsec algorithm.
+
+Configurable Parameters:
+....................................................................................
+Name                    Type     Optional   Comment                         Default
+N-Integration-Samples   int      No         Maximum integration iterations  10000
+NuE-XSecRelError        double   No                                         0.001
```

Comment:
Same comment as before

Reply:
deleted this fcl since it's an interface